test/test_search.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+PARENT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, PARENT)
```

Comment:
Just to run it without installing the package, but you're right this should be adapted and added to the current tests